src/org/opensolaris/opengrok/configuration/Group.java

```diff
@@ -37,6 +37,7 @@
 
     private String name;
     private String pattern = "";
+    private Pattern compiledPattern = Pattern.compile("(" + this.pattern + ")");
```

Comment:
okay, could you add a comment there so that's the intent is clear ? (basically what you've written above)